packages/atomic/src/components/commerce/atomic-product-multi-value-text/atomic-product-multi-value-text.ts

```diff
@@ -0,0 +1,213 @@
+import {withTailwindStyles} from '@/src/decorators/with-tailwind-styles';
+import {getFieldValueCaption} from '../../../utils/field-utils';
+import {titleToKebab} from '../../../utils/utils';
+import type {CommerceBindings} from '../atomic-commerce-interface/atomic-commerce-interface';
+import styles from './atomic-product-multi-value-text.tw.css';
+
+/**
+ * @alpha
+ *
+ * The `atomic-product-multi-value-text` component renders the values of a multi-value string field.
+ *
+ * @part product-multi-value-text-list - The list of field values.
+ * @part product-multi-value-text-separator - The separator to display between each of the field values.
+ * @part product-multi-value-text-value - A field value.
+ * @part product-multi-value-text-value-more - A label indicating some values were omitted.
+ * @slot product-multi-value-text-value-* - A custom caption value that's specified for a given part of a multi-text field value. For example, if you want to use `Off-Campus Resident` as a caption value for `Off-campus apartment` in `Off-campus apartment;On-campus apartment`, you'd use `<span slot="product-multi-value-text-value-off-campus-apartment">Off-Campus Resident</span>`). The suffix of this slot corresponds with the field value, written in kebab case.
+ */
+@customElement('atomic-product-multi-value-text')
+@bindings()
+@withTailwindStyles
+export class AtomicProductMultiValueText
+  extends LitElement
+  implements InitializableComponent<CommerceBindings>
+{
+  static styles = [unsafeCSS(styles)];
+
+  @state()
+  bindings!: CommerceBindings;
+
+  @state()
+  error!: Error;
+
+  public breadcrumbManager!: BreadcrumbManager;
+
+  private productController = createProductContextController(this);
+
+  /**
+   * The field that the component should use.
+   * The component will try to find this field in the `Product.additionalFields` object unless it finds it in the `Product` object first.
+   * Make sure this field is present in the `fieldsToInclude` property of the `atomic-commerce-interface` component.
+   */
+  @property({reflect: true})
+  public field!: string;
+
+  /**
+   * The maximum number of field values to display.
+   * If there are _n_ more values than the specified maximum, the last displayed value will be "_n_ more...".
+   */
+  @property({reflect: true, type: Number, attribute: 'max-values-to-display'})
+  public maxValuesToDisplay = 3;
+
+  /**
+   * The delimiter used to separate values when the field isn't indexed as a multi value field.
+   */
+  @property({reflect: true})
+  public delimiter: string | null = null;
+
+  @state() private values: string[] = [];
+
+  public initialize() {
+    let searchOrListing: ProductListing | Search;
+
+    if (this.bindings.interfaceElement.type === 'product-listing') {
+      searchOrListing = buildProductListing(this.bindings.engine);
+    } else {
+      searchOrListing = buildSearch(this.bindings.engine);
+    }
+
+    this.breadcrumbManager = searchOrListing.breadcrumbManager();
```

Comment:
When you can, do prefer using const over let (or var, tho rarely if ever you should use var nowadays).

```suggestion
  private get isInSearchInterface() {
    return this.bindings.interfaceElement.type === 'search';
  }

  public initialize() {
    const interfaceElement = this.isInSearchInterface ?
      buildSearch(this.bindings.engine) :
      buildProductListing(this.bindings.engine);
    this.breadcrumbManager = interfaceElement.breadcrumbManager();
```